wiki/whitelist-process.md

```diff
@@ -115,10 +115,12 @@ around once every 24 hours. The scrapers and the increase in users can potential
 Secondly, this can potentially incentivize more cheating because people now can earn money from credits. However it also incentivises people to try and catch cheating since they would be losing out on the Gridcoin gained by the cheater 
 
 ## What's the whitelisting process?
-The first step is making sure you as the admin of the project want to be whitelisted. If you, somewhere public you should say that you were interested and a date when they would want this to happen (anywhere that someone could check such as project forums, Reddit, etc). Following this, find someone to make a poll for you on Gridcoin. Lastly if the community votes votes "yes", then you
-get whitelisted.
 
-## Some things you might want to do beforehand: 
+1. The first step is making sure you as the admin of the project want to be whitelisted. You should state somewhere publicly that you are interested and a date when they would want this to happen (anywhere that someone could check such as project forums, Reddit, etc). 
+2. Find someone to make a poll for you on Gridcoin (or make one yourself)
+3. If the community votes "yes", then you get whitelisted. So far every project put to a whitelist poll has been whitelisted, our users are enthusiastic about contributing to science!
```

Comment:
> So far every project put to a whitelist poll has been whitelisted

This is not strictly true. In 2017, Xansons was not voted to be whitelisted (source: https://github.com/Erkan-Yilmaz/GRC/issues/11)

Reply:
fixed